src/services/nginx/default.conf

```diff
@@ -6,6 +6,10 @@ upstream broker {
     server broker:8000;
 }
 
+upstream openfinconfig {
+    server openfinconfig:8080;
+}
+
```

Comment:
Can we please make the paths `/openfin/app.sjon` and `/openfin/launcher.json`. I think having `config` in there is redundant.